docs/figma/ios_prototype_v2/styles.css

```
@@ -0,0 +1,844 @@
+  --pp-surface-strong: rgba(255, 255, 255, 0.14);
+  --pp-border: rgba(188, 204, 220, 0.28);
+  --pp-text: #f8fbff;
+  --pp-text-muted: rgba(240, 244, 248, 0.74);
+  --pp-text-soft: rgba(240, 244, 248, 0.58);
+  --pp-shadow: 0 18px 40px rgba(8, 15, 31, 0.35);
+  --pp-shadow-soft: 0 10px 24px rgba(8, 15, 31, 0.18);
+  --pp-radius-xl: 30px;
+  --pp-radius-lg: 24px;
+  --pp-radius-md: 18px;
+  --pp-radius-sm: 14px;
+  --pp-font: "SF Pro Display", "SF Pro Text", -apple-system, BlinkMacSystemFont, sans-serif;
```

Stylelint (17.4.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)